ui/src/app/components/error-handler.tsx

```diff
@@ -127,9 +127,10 @@ export const ErrorHandler = fp.flow(withUserProfile(), withGlobalError())(class
   render() {
     const {globalError} = this.props;
     const {serverDownStatus: {apiDown, firecloudDown, notebooksDown}} = this.state;
-
     return globalError !== undefined && <React.Fragment>
-      {globalError.statusCode === 500 && <div style={styles.errorHandler}>
+      {/* TODO(RW-RW-4392): this component should be revamped to be more generic, allowing display
+      of arbitrary error messages as well as response code based messaging. */}
+      {[400, 500].includes(globalError.statusCode) && <div style={styles.errorHandler}>
```

Comment:
Why are we adding 400s here? I think we would want BadRequests to be handled differently?